docs/kafka/getting-started-kafka/README.adoc

```diff
@@ -185,10 +185,10 @@ image::sak-kafka-overview.png[Image of initial empty instances table]
 ////
 
 [id="proc-creating-service-account_{context}"]
-== Creating a service account to connect to a Kafka instance in {product-kafka}
+== Creating a service account to connect to a Kafka instance in {product-long-kafka}
 
 [role="_abstract"]
-To connect your applications or services to a Kafka instance in the {product-kafka} web console, you need to create a service account that's associated with the instance. You also need to save the bootstrap server endpoint for the instance, the generated service account credentials, and the authentication token endpoint to a secure location. You'll use the service account and Kafka instance information later when you configure your application.
+To connect your applications or services to a Kafka instance in {product-long-kafka}, you need to create a service account that's associated with the instance. You also need to save the bootstrap server endpoint for the instance, the generated service account credentials, and the authentication token endpoint to a secure location. You'll use the service account and Kafka instance information later when you configure your application.
 
 .Prerequisites
 * You've created a Kafka instance and the instance is in *Ready* state.
```

Comment:
Suggest adding 'the' before Ready, in line with other recent updates.

Reply:
> LGTM. Just added a comment on a future update to be made during the full sweep of the GS guide as it caught my eye.

Fully agree. That one has actually been a big bugbear of mine. Might as well nobble it now!